apps/extension/src/ui/domains/Portfolio/AssetsTable/usePortfolioSymbolBalances.ts

```diff
@@ -93,11 +94,11 @@ const sortSymbolBalancesBy =
 
 export const usePortfolioSymbolBalances = (balances: Balances) => {
```

Comment:
As far as I can tell, this hook is used exclusively to populate the list of tokens shown in the Popup and Dashboard UIs.

It doesn't affect the totals calculations.